src/docs/sphinx/blueprint_mesh.rst

```diff
@@ -504,12 +516,85 @@ The following diagram illustrates a simple **multi-buffer** material set example
       # +--------+--------+--------+
       #
 
+      matsets:
+        matset:
+          topology: topology
+          volume_fractions:
+            a:
+              values: [0, 0, 0, a1, 0, a0]
+              indices: [5, 3]
+            b:
+              values: [0, b0, b2, b1, 0]
+              indices: [1, 3, 2]
+
+
+Material Set Indexing Variants
+=================================
+
+Another dimension along which material sets can vary is in how their volume fractions/topological element association is established.
+This dimension produces two additional material set variants: **element-dominant** association (element indices control volume fraction order) and **material-dominant** association (material indices control volume fraction order).
```

Comment:
Similar to the previous sentence, I'd rephrase to something like "Material sets can have **element-dominant** association (volume fractions for each element are grouped in the data buffer) or **material-dominant** association (volume fractions for eachmaterial are grouped in the data buffer)."